Static/DataSource.swift

```diff
@@ -188,11 +187,11 @@ extension DataSource: UITableViewDataSource {
     }
 
     public func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return rowForIndexPath(indexPath)?.height ?? 44
+        return rowForIndexPath(indexPath)?.height ?? UITableViewAutomaticDimension
     }
     
     public func tableView(tableView: UITableView, estimatedHeightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return rowForIndexPath(indexPath)?.height ?? 44
+        return rowForIndexPath(indexPath)?.height ?? UITableViewAutomaticDimension
```

Comment:
Estimate of 44 sounds good (esp since it’s maintaining the behavior from the previous release). If we want to make an improvement for the scroll indicator (which I think is nice to have) we should talk about it in a separate issue and potentially add that in later since our current release is very broken and we should get this out as soon as we can. :octocat: